hr_360_view/wizard/hr_employee_search_wizard.py

```diff
@@ -134,14 +135,16 @@ def name_get(self):
 
     @api.multi
     def search_jobseeker(self):
+        now_year = str(datetime.now().year)
+        before_year = str(datetime.now().year - 100)
```

Comment:
before_year is a pretty ambiguous variable name, consider changing to last_century or something more descriptive.